packages/data/src/api/dynamicNftMetadata/README.md

```diff
@@ -18,7 +18,7 @@ Example usage:
 import { dynamicNftMetadata } from  '@mintbase-js/data'
 
 const props = {
-  contractAddress: 'dynamic.mintbase2.near',
+  contractAddress: 'dynamic.mintbasev2.near',
```

Comment:
This appears so many times it could easily be a global constant. Then future changes wouldn't touch so much code.

Reply:
It already is a global constant, but used only sporadically. The changes also resulted from the fact that the name was arbitrarly used in places and I just replaced it in all the project even where it didn't have anything to do with the factory.